server/models/Group.js

```diff
@@ -56,6 +56,15 @@ Group.associate = models => {
     through: models.GroupUser,
     foreignKey: 'groupId',
   });
+  Group.addScope('defaultScope', () => ({
+    include: [
+      {
+        model: models.GroupUser,
+        as: 'groupMemberships',
+        required: false,
+      },
+    ],
+  }));
```

Comment:
🍖 — I am a little worried about perf, this seems like it could get quite large

Reply:
Agreed, seems like it's unsustainable

Reply:
we'd have to prefetch the count at least, gonna try taking that approach outlined here https://stackoverflow.com/questions/37817808/counting-associated-entries-with-sequelize